### ISSUE TITLE
Complete Parameter Explainers

### ISSUE THREAD

**Issue by federicoast** [kleros/gtcr#85]
Here we need to complete the missing hover explainers.

@zambian can you please write these little hover explainers? You can take the information from the blog post William wrote about parameters. It's on the blog. Also ask William in case there's any doubt.

We need hover explainers on the following items:

- Deposit

Current wording says: "The deposit is the sum of the challenge reward bounty and the arbitration cost."

This is more or less meaningless for the user.

It should say something like: "This is the sum that will have to be... If you set this low, then X happens. If you set this high, then Y happens".

The user needs to be aware of the trade-offs here.

- Court

We need a hover here saying: "This is the court where disputes between submitters and challengers will be adjudicated."

Then we want to give the user some guidance of how to select the right court for his use case. 

- Number of Jurors

We need to write a hover explainer for Number of Jurors.

It should say something like: "This is the number of jurors that will be drawn to adjudicate disputes. If you set the number low, then X happens. If you set the number high, then Y happens".

- Arbitration Cost

We need to add a hover explaining what the number of Arbitration Cost means. Is it a good thing or a bad thing that it's higher or lower?

![FireShot Capture 1043 - Kleros · Curate - https___curate kleros io_factory](https://user-images.githubusercontent.com/16290963/83655372-ba720980-a594-11ea-8e9c-e2d9dcd472b8.jpg)

**Comment by zambian** [comment]
Yes, I will have all this content from the explainer. It should be cross compatible.
